app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -1,5 +1,6 @@
 package org.simple.clinic.overdue
 
+import androidx.annotation.VisibleForTesting
```

Comment:
Seems like this import was part of the previous commit.

Reply:
Good catch

Reply:
We don't need to rebase this as we are going to squash the commits anyway. So, it's fine. We can go ahead and merge this in.